bin.src/gen2to3.py

```diff
@@ -36,6 +36,7 @@ def main(config=None):
     instrument = HyperSuprimeCam()
     convertRepoConfig = ConvertRepoTask.ConfigClass()
     instrument.applyConfigOverrides(ConvertRepoTask._DefaultName, convertRepoConfig)
+    convertRepoConfig.storageClasses["jointcal_wcs"] = "Wcs"
```

Comment:
It would be better to put this in the config defaults for `ConvertRepoTask` [here|https://github.com/lsst/obs_base/blob/master/python/lsst/obs/base/gen2to3/convertRepo.py#L131], since it's not specific to ci_hsc.